lib/uinta/plug.ex

```diff
@@ -109,7 +111,13 @@ if Code.ensure_loaded?(Plug) do
         ignored_paths: Keyword.get(opts, :ignored_paths, []),
         include_unnamed_queries: Keyword.get(opts, :include_unnamed_queries, false),
         include_variables: Keyword.get(opts, :include_variables, false),
-        filter_variables: Keyword.get(opts, :filter_variables, @default_filter)
+        filter_variables: Keyword.get(opts, :filter_variables, @default_filter),
+        success_log_sampling_percent:
```

Comment:
Would ratio be clearer than percent if it's a float?

Reply:
yea I was debating . I ll change it to ratio